src/pe_reports/report_generator.py

```diff
@@ -1,17 +1,38 @@
-"""A tool for creating Posture & Exposure reports."""
+"""A tool for creating Posture & Exposure reports.
+
+Usage:
+    pe-reports REPORT_DATE DATA_DIRECTORY OUTPUT_DIRECTORY
+
+Arguments:
+  REPORT_DATE       Date of the report, format YYYY-MM-DD.
+  DATA_DIRECTORY    The directory where the excel data files are located.
+                    Organized by owner.
+  OUTPUT_DIRECTORY  The directory where the final PDF reports should be saved.
+
+Options:
+  -h --help     Show this message.
+  -v --version  Show version information.
+  --log-level=LEVEL     If specified, then the log level will be set to
+                        the specified value.  Valid values are "debug", "info",
+                        "warning", "error", and "critical". [default: info]
```

Comment:
This formatting should be consistent. There should be two spaces between the longest option and its description and every other option should be spaced to match.

```suggestion
Options:
  -h --help          Show this message.
  -v --version       Show version information.
  --log-level=LEVEL  If specified, then the log level will be set to
                     the specified value.  Valid values are "debug", "info",
                     "warning", "error", and "critical". [default: info]
```